crates/tracing/Cargo.toml

```diff
@@ -37,7 +37,7 @@ opentelemetry-otlp = { workspace = true, features = [
     "logs",
     "trace",
     "metrics",
-    "reqwest-client",
+    "reqwest-rustls",
```

Comment:
```suggestion
```

Looks like should be able to use https://docs.rs/opentelemetry-otlp/0.16.0/opentelemetry_otlp/struct.HttpExporterBuilder.html#method.with_http_client to configure our HTTP client from `core`